File: Python3/26. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        if len(nums) == 1 or len(nums) == 0:
            return len(nums)
        
        cur = 0
        
        for i in range(len(nums)-1):
            if nums[i] == nums[i+1]:
                continue
            nums[cur] = nums[i]
            cur += 1
        
        nums[cur] = nums[len(nums)-1]
        
        return cur+1

File: Java/26. Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length == 0 || nums.length == 1){
            return nums.length;
        }
        
        int cur = 0;
        for(int i=0; i<nums.length-1; i++){
            if(nums[i] == nums[i+1]){
                continue;
            }
            
            nums[cur] = nums[i];
            cur++;
        }
        
        nums[cur] = nums[nums.length-1];
        
        return cur+1;
    }
}